zulip_bots/zulip_bots/bots/flock/flock.py

```diff
@@ -39,11 +39,11 @@ def get_recipient_id(recipient_name: str, config: Dict[str, str]) -> Tuple[Optio
     payload = {
         'token': token
     }
-    res, error = make_flock_request(USERS_LIST_URL, payload)
-    if res is None:
+    users, error = make_flock_request(USERS_LIST_URL, payload)
```

Comment:
I suppose the commits renaming res to users could be squashed into one.

Reply:
@aero31aero Squashed these commits into one.